apps/rehabstod/src/pages/CurrentSickLeaves/components/PrintTable.tsx

```diff
@@ -27,20 +27,22 @@ export function PrintTable({ sickLeaves, showPersonalInformation }: { sickLeaves
     return null
   }
 
+  const sortedList = sortTableList(sickLeaves, getSickLeavesColumnData)
+
   return (
     <div>
       {sortTableList(sickLeaves, getSickLeavesColumnData)?.map((sickLeave) => (
```

Comment:
This should be replaced this with the new `sortedList` so that we don't have to sort the list twice :)